python_modules/dagster/dagster/core/definitions/schedule_definition.py

```diff
@@ -223,6 +224,7 @@ def __init__(
                     solid_selection, "solid_selection", of_type=str
                 ),
             )
+            self._job = None
```

Comment:
Unsure what the behavior should be in this case - SensorDefinition has a `targets` attribute that captures both job and pipeline info, should we do something similar here?

Reply:
I think you just hard error if someone tries to access the job attribute and didn't provide a job on their sensor.